File: online-shop/online-shop-backend/src/main/java/com/example/onlineshop/security/models/Product.java
package com.example.onlineshop.security.models;

import lombok.Data;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;


import javax.persistence.*;

@Getter
@Setter
@Entity
@RequiredArgsConstructor
public class Product {

    @Id
    @GeneratedValue(strategy=GenerationType.AUTO)
    private Long id;

    @Column(name = "sketching")
    private String sketching;

    @Column(name = "finalProduct")
    private String finalProduct;

    @Column(name = "productName")
    private String productName;

    @Column(name = "price")
    private Integer price;

    @Column(name = "description")
    private String description;

}

File: online-shop/online-shop-frontend/src/app/models/product.ts
import {Designer} from "./designer";
import { User } from "./user";

export class Product {

  id: number = 0;
  sketching:string ='';
  finalProduct:string ='';
  productName:string ='';
  price:number=0;
  description:string ='';
  designer: Designer = new Designer();
  status:string=''
  user: User = new User();
  constructor(){
  }

}
